### ISSUE TITLE
Reabrir ticket #4

### ISSUE THREAD

**Issue by gildonei** [PedroBegosso/filter_results#8]
Oi Pedro, estou tendo o mesmo problema comentado no Ticket #4 porém vou complementar com algumas informações adicionais

1 - Está acontecendo apenas em localhost
2 - Acontece com mais frequencia no Google Chrome
3 - Quando acontece, todas as tags Script e CSS são inseridas logo após a tag "body"
4 - CakePHP 2.3.5 (último stable release)
5 - Windows 7 (com todos os updates)
6 - Rodando Zendserver Pro v5.6.0

Pontos interessantes observados
O "bug" acontece apenas em uma ou outra página, vamos supor, /produtos/imagens acontece e em /produtos/categorias não acontece, mesmo utilizando o mesmo padrão de view, duplicando o código de uma view para outra isso vem a acontecer
![erro-white-space-1](https://f.cloud.github.com/assets/1370979/542131/a957683c-c21d-11e2-9d58-7604e485ea0f.png) - Imagem com o "bug" 

![sem-erro-white-space-1](https://f.cloud.github.com/assets/1370979/542152/025fdf90-c21e-11e2-9399-fafa8b26904d.png) - Imagem sem o "bug"

Observando que ambos os controllers utilizam o filter results e possuem filtros similares, é uma coisa "muito esporádica"

Em outras pesquisas no Google, vi coisas similares acontecendo com WordPress, e outros Frameworks.

Acho q este tópico poderia ficar aberto e tentarmos descobrir e ajudar a comunidade quando acharmos a resposta.

**Comment by PedroBegosso** [comment]
@gildonei como faz muito tempo que o ticket esta aberto, e ja atualizei bastante o plugin. 
Pergunto: o problema persiste? ^^'

**Comment by gildonei** [comment]
Sim, ele persiste, já fiz várias outras tentativas de correção sem sucesso, não sei exatamente o que poderia estar ocasionando o problema.

**Comment by gildonei** [comment]
OI Pedro acho q finalmente depois de muito tempo identifiquei o motivo do problema estar acontecendo.

Isto estava acontecendo porque o arquivo FilterFormHelper.php estava codificado como UTF-8 (com BOM) o que fazia inserir o "caracter invisível" no documento.

Anexo envio imagem do caracter especial que aparecia antes do doctype no documento.

Não sei se isto já foi corrigido nas versões mais novas, porém na versão que uso do Plugin em alguns projetos tive de alterar e resolveu.

![utf-bom-char](https://cloud.githubusercontent.com/assets/1370979/4224588/7f5f09a0-3926-11e4-80cc-8de621d0b84b.png)

Abraços.
